grunt/release.js

```diff
@@ -1,5 +1,7 @@
 module.exports = {
   options: {
+    changelog: true,
+    changelogText: '<%= version %>\n',
```

Comment:
I'd say the default `changelogText` is better than this: https://github.com/geddski/grunt-release#options